### ISSUE TITLE
can't show blocks when set horizontalLayout to true

### ISSUE THREAD

**Issue by shenghuowang** [LLK/scratch-gui#1059]
### Expected Behavior

_Please describe what should happen_

### Actual Behavior

_Describe what actually happens_

### Steps to Reproduce

I've set horizontalLayout to true in scratch-blocks Configuration,but after doing this,no blocks in the Blocks category

### Operating System and Browser

_e.g. Mac OS 10.11.6 Safari 10.0_

**Comment by Kenny2github** [comment]
Hi there! The horizontal layout is not yet being worked on - the team is focusing on the vertical layout first.

**Comment by shenghuowang** [comment]
Thanks a lot,any plan to support horizontal layout?

**Comment by Kenny2github** [comment]
Yes, absolutely! However, horizontal layout will start development after the team as confirmed that the vertical layout is up and working just like Scratch 2.0 - so I’m afraid you’ll have to wait 😕

**Comment by rachel-fenichel** [comment]
While all horizontal blocks can be rendered vertically, not all vertical blocks can be rendered horizontally.  If you want to experiment with the horizontal blocks you can do that with the [horizontal playground](http://llk.github.io/scratch-blocks/playgrounds/tests/horizontal_playground.html) in scratch-blocks.

**Comment by m93a** [comment]
What is the status of this bug? I would like to experiment with horizontal layout, but I'm unable to get it working in the GUI.

I've added `horizontalLayout: true` to the Blocks configuration ([at this line](https://github.com/LLK/scratch-gui/blob/develop/src/containers/blocks.jsx#L96)) and all the blocks disappeared. Is there anything else I can do to get at least some blocks working?
